dags/monitoring/events.py

```diff
@@ -56,11 +55,22 @@
 )
 
 
+def short_circuit_on_no_new_tables(task_id):
+    def _do_short_circuit(**kwargs):
+        """Short circuit if no new tables"""
+        ti = kwargs['ti']
+        xcom_result = ti.xcom_pull(task_ids=task_id)
+        is_xcom_none_or_empty = not xcom_result or not xcom_result.get('new_tables', [])
+        return not is_xcom_none_or_empty
```

Comment:
return not xcom_result and not xcom_result.get('new_tables', [])

Reply:
> return not xcom_result and not xcom_result.get('new_tables', [])

There's an extra `not` in line 64 that will be missed this way.

Reply:
it should not 
not (not a or not b) == not a and not b